tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-indexed-access.bal

```diff
@@ -88,6 +88,83 @@ function testLengthOfXMLSequence() returns [int, int, int, int] {
     return [x1.length(), x2.length(), x2[2].length(), x3.length()];
 }
 
+const i = -1;
+
+function testInvalidXMLAccessWithNegativeIndex() returns error? {
```

Comment:
Shall we also add several  xm subtype as sequence tests. `eg: xml<xml:Comment>` and so on

Reply:
Added

Reply:
You have added `xml:Comment` which is xml singelton type tests not `xml<xml:Comment>` which is xml  sub type sequence tests.